sklearn/tree/export.py

```diff
@@ -120,7 +120,8 @@ def export_graphviz(decision_tree, out_file="tree.dot", max_depth=None,
 
     proportion : bool, optional (default=False)
         When set to ``True``, change the display of 'values' and/or 'samples'
-        to be proportions and percentages respectively.
+        to be proportions and percentages respectively. When set to 'both',
```

Comment:
Oh the argument `proportion=both` seems a bit weird. But I don't really have better thoughts.